opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -11,7 +11,7 @@ use std::{borrow::Cow, fmt::Debug};
 
 /// `LogExporter` defines the interface that log exporters should implement.
 #[async_trait]
-pub trait LogExporter: Send + Debug {
+pub trait LogExporter: Send + Sync + Debug {
```

Comment:
Please add an entry to changelog.md file.

Reply:
Done!